pkg/target/app_dto.go

```diff
@@ -64,5 +64,10 @@ func (app *Application) createCommoditiesSold() ([]*proto.CommodityDTO, error) {
 	appComm, _ := CreateTransactionCommodity(app.UUID, &(app.QPS), proto.CommodityDTO_TRANSACTION)
 	result = append(result, appComm)
 
+	if app.ResponseTime != nil {
```

Comment:
In supplyChain definition, Application will sell `ResponseTime` . If it does not sell it, then the supplyChain check may fail.